market/settings.py

```diff
@@ -109,10 +111,10 @@
 MEDIA_URL = '/media/'
 
 STATIC_URL = '/static/'
-STATIC_ROOT = os.path.join(BASE_DIR, 'static/')
+# STATIC_ROOT = os.path.join(BASE_DIR, 'static/')
```

Comment:
@doradomiguel35 do not remove my configurations, it works perfectly and should work as is.